src/test/java/com/mongodb/kafka/connect/source/MongoCopyDataManagerTest.java

```diff
@@ -436,6 +426,22 @@ private void sleep() {
     sleep(500);
   }
 
+  private static String createTemplate(final int id) {
+    return createTemplate(id, "myDB", "myColl");
+  }
+
+  private static String createTemplate(final int id, final String dbName, final String collName) {
+    return format(CHANGE_STREAM_DOCUMENT_TEMPLATE, id, dbName, collName, id, id, id);
+  }
+
+  private static RawBsonDocument createInput(final String json) {
+    return RawBsonDocument.parse(format(json, "__"));
```

Comment:
Should `MongoCopyDataManager.NAMESPACE_FIELD` be used here? (requires making the field package-private)

Reply:
Ack will update